# This message signals a change in an Entity-Component pair, as specified by
# the ECData part

ECData data

int8 ADDED=1
int8 UPDATED=2
int8 REMOVED=3

int8 updateType
